service/src/test/java/bio/terra/pipelines/dependencies/sam/SamServiceTest.java

```diff
@@ -80,10 +80,11 @@ void checkHealthWithException() throws ApiException {
 
   @Test
   void getServiceAccountToken() throws IOException {
+    GoogleCredentials mockCredentials = GoogleCredentials.create(new AccessToken("hi", null));
+
     try (MockedStatic<GoogleCredentials> utilities = Mockito.mockStatic(GoogleCredentials.class)) {
       SamClient samClient = mock(SamClient.class);
-      GoogleCredentials googleCredentials = new GoogleCredentials(new AccessToken("hi", null));
```

Comment:
this method of creating credentials is deprecated now

Reply:
ooo fancy 😮